app/app/users/[user_id]/page.tsx

```diff
@@ -0,0 +1,87 @@
+"use client";
+import Card from "react-bootstrap/Card";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Spinner from "react-bootstrap/Spinner";
+import Table from "react-bootstrap/Table";
+import AppBreadCrumb from "@/components/AppBreadCrumb";
+import { useToken } from "@/utils/auth";
+import { useUser } from "@/utils/users";
+import { User } from "@/types/users";
+import { formatDateTime } from "@/utils/formatters";
+
+type UserColumn = {
+  name: keyof User;
```

Comment:
I cant decide if I'd prefer this to be called `columnName` since there is a column named Name and I was getting confused, but maybe then it would be overkill with `user[col.columnName]`

Reply:
👍 i prefer `name`, but am open to alternatives. `key` or `field` also come to mind. whatever we choose, we should use the same property name to describe a column name in our graphql queries. which is `name` at the moment: https://github.com/cityofaustin/vision-zero/blob/a64f5f72a87a7eac726daf916a63b305eb7c21f4/app/types/types.ts#L15

☝️ the typing on these column configs is a bit messy and i kinda want to take a second pass at it once we remove the zod stuff.

Reply:
i think i prefer `field` so as not to get confused with something like chias example